gtsfm/utils/triangulation.py

```diff
@@ -0,0 +1,41 @@
+"""Utilities to support triangulation.
+
+Authors: Ayush Baid
+"""
+import numpy as np
+from gtsam import PinholeCameraCal3Bundler
+
+
+def calculate_triangulation_angle_in_degrees(
+    camera_1: PinholeCameraCal3Bundler, camera_2: PinholeCameraCal3Bundler, point_3d: np.ndarray
+) -> float:
+    """Calculates the angle formed at the 3D point by the rays backprojected from 2 cameras.
+
+    Args:
+        camera_1: the first camera.
+        camera_2: the second camera.
+        point_3d: the 3d point to compute the angle at.
```

Comment:
"to compute the angle at" is a bit vague. maybe we can add a bit more detail

Reply:
does this look better?